src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/ExceptionAssert.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using Xunit;
+
+namespace BasicEventSourceTests
+{
+    /// <summary>
+    /// Allows testing for multiple exceptions in one test method (as opposed to the ExceptionExpectedAttribute)
+    /// </summary>
+    public static class ExceptionAssert
+    {
+        /// <summary>
+        /// Validates that an exception of type T is thrown. If no or another exception is thrown the test will fail.
+        /// </summary>
+        /// <typeparam name="T">The expected exception type.</typeparam>
+        /// <param name="testMethod">The test to execute.</param>
+        public static void Throws<T>(Action testMethod) where T : Exception
```

Comment:
Can't we just use xunit's ```Assert.Throws<T>(Action)```?  That's what we use elsewhere in the corefx codebase.